tests/RoyalmailTest.php

```diff
@@ -185,4 +424,67 @@ public function testRoyalMailClassNull()
         $this->assertEquals($this->emptyArray, $this->calculateMethodClass->getMethods(null, null, null));
     }
 
+    /**
+     * Test co compare the meta names vs the clean name to check that the correct value
+     * exists and is being used.
+     */
+    public function testMethodToMetaVsCleanName()
+    {
+        foreach ($this->dataClass->mappingMethodToMeta as $array => $data){
+            foreach ($this->dataClass->mappingCleanNameToMethod as $method => $methodData)
+            {
+                if ($data[0] ==  $methodData[2]){
+                    $this->assertEquals($data[4], $methodData[1]);
+                }
+            }
```

Comment:
Could add in a flag her to through an error if it gets to the end of the for loop and doesn't find a clean name.  Or is that already tested somewhere else?